SCF-Assignment-5/Question3/NQueenTest.java

```diff
@@ -0,0 +1,22 @@
+package Question3;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NQueenTest {
+
+	@Test
+	public void testPositive() {
+		int[][] board = new int[4][4];
+		boolean result = NQueensProblem.nQueen(board, 0, 4);
+		assertEquals(result, true);

```

Comment:
We can use assertTrue and assertFalse for checking boolean results instead of assertEquals